nautobot/ipam/tables.py

```diff
@@ -410,8 +410,8 @@ class PrefixDetailTable(PrefixTable):
     class Meta(PrefixTable.Meta):
         fields = (
             "pk",
-            "namespace",
             "prefix",
+            "namespace",
```

Comment:
Was there an open issue regarding this ordering? I thought there was but I can't find it.

Reply:
I did too and I also can't find it.

Reply:
Got it now #3894